src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drv.c

```diff
@@ -1252,7 +1252,7 @@ static void xocl_cma_mem_free(struct xocl_dev *xdev, uint32_t idx)
 
 	if (cma_mem->regular_page) {
 		dma_unmap_page(&xdev->core.pdev->dev, cma_mem->paddr,
-			cma_mem->size, PCI_DMA_BIDIRECTIONAL);
+			cma_mem->size, DMA_BIDIRECTIONAL);
```

Comment:
Curious. Where did the `PCI_DMA_BIDIRECTIONAL` go?

Reply:
`PCI_DMA_BIDIRECTIONAL` was part of [`pci-dma-compat.h`](https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/include/linux/pci-dma-compat.h?h=linux-5.17.y). This header wraps DMA operations but [does that in an inconsistent way according to a kernel developer](https://lore.kernel.org/kernel-janitors/alpine.DEB.2.22.394.2007120902170.2424@hadrien/). I am not well-versed with kernel dev, so @houlz0507 may give a better explanation.

`PCI_DMA_BIDIRECTIONAL` was anyway pointing to `DMA_BIDIRECTIONAL`, so Linux got rid of the wrapper.